Added the UI Feature Update in the Password Visibility Toggle

VarunT11 opened the pull request:
**Issue Reference** : https://github.com/odk-x/tool-suite-X/issues/200

**Small Description of what this PR does** : 
- Adds the Android Support Design Dependency in the **_build.gradle_** File of the App Module of the Project.
- Removes the _Checkbox_ from the Layout XML File of the **_login_fragment.xml_** and **_password_dialog_layout.xml_** Files and changes the normal _EditText_ to _TextInputEditText_ provided by the Android Support Dependency.
- Removed the use of the _Checkbox_ from the _**LoginFragment.java**_ and _**PasswordDialogFragment.java**_ file.

**Screenshots** :
![Screenshot_20210313-122355](https://user-images.githubusercontent.com/58032608/111080204-62a0ff80-8523-11eb-80fc-6f142aa57dde.jpg)
![Screenshot_20210313-122412](https://user-images.githubusercontent.com/58032608/111080222-76e4fc80-8523-11eb-8855-7d571ccf328f.jpg)
![Screenshot_20210313-124715](https://user-images.githubusercontent.com/58032608/111080224-79dfed00-8523-11eb-8dd2-15e998f98891.jpg)
![Screenshot_20210313-124701](https://user-images.githubusercontent.com/58032608/111080226-7d737400-8523-11eb-93b6-cee6a09375a0.jpg)

wbrunette commented:
runtests

linl33 commented on services_app/build.gradle:
```diff
@@ -122,6 +122,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.preference:preference:1.1.1'
     implementation 'androidx.fragment:fragment:1.3.0-beta01'
+    implementation 'com.android.support:design:28.0.0'
```
Please update this library, see https://material.io/develop/android/docs/getting-started#new-namespace-and-androidx

linl33 reviewed (changes requested):
@VarunT11 Thanks for implementing this feature!

The new password field works great, just a few comment
1. Please remove the now unused `import`s in the fragment classes
2. Could you bring the changes to `password_challenge_dialog_layout` as well? That layout is used to enter the admin password.

VarunT11 commented:
**Small Description of Changes Performed**:
- Added the Password Visibility Toggle Feature to the _**AdminPasswordChallengeFragment.**_
- Removed the Unused Imports from _**LoginFragment**_ , _**PasswordDialogFragment**_ and _**AdminPasswordChallengeFragment.**_
- Updated the _**build.gradle**_ File by updating the Libraries.

**Screenshots**:
![Screenshot_20210315-124801__01](https://user-images.githubusercontent.com/58032608/111118672-1ab7c200-858f-11eb-887e-7221f3f44d29.jpg)
![Screenshot_20210315-124806__01](https://user-images.githubusercontent.com/58032608/111118680-1be8ef00-858f-11eb-80e1-75ebd4a28be3.jpg)

**PS**: @linl33 The Requested Changes have been performed. Please review them!

linl33 commented:
runtests
